File: front/small-box/src/app/models/depositControlDto.ts
export class DepositControlDto {
    id!: number
    place!: string;

    itemDescription!: string;

    itemCode!: string;
    quantity!: number;
    expirationDate!:Date;
    provider!: string;

    measureUnit!: string;
    itemUnitPrice!:number;
    itemTotalPrice!:number;
}

File: src/main/java/com/lord/small_box/dtos/DepositControlDto.java
package com.lord.small_box.dtos;

import java.math.BigDecimal;
import java.util.Calendar;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class DepositControlDto {
    private Long id;
    
	private String place;
	
	private String itemName;
	
	private String itemCode;
	
	private int quantity;
	
	private Calendar expirationDate;
	
	private String provider;
	
	private String measureUnit;
	
	private BigDecimal itemUnitPrice;
	
	private BigDecimal itemTotalPrice;
}
